metrics/sources/manager.go

```diff
@@ -164,12 +164,16 @@ responseloop:
 func scrape(s MetricsSource, start, end time.Time) (*DataBatch, error) {
 	sourceName := s.Name()
 	startTime := time.Now()
-	defer lastScrapeTimestamp.
-		WithLabelValues(sourceName).
-		Set(float64(time.Now().Unix()))
-	defer scraperDuration.
-		WithLabelValues(sourceName).
-		Observe(float64(time.Since(startTime)) / float64(time.Microsecond))
+	defer func() {
```

Comment:
Shouldn't be the begging of the scraping?

Reply:
I wanted it to be consistent with other `last` metrics

Reply:
@crassirostris Should `last` event timestamp come from before or after execution?

Reply:
Making it consistent with `metrics/sources/manager.go` seems reasonable

Ambiguity of the description is another problem, but it's outside of this PR's scope